README.md

```diff
@@ -309,7 +309,7 @@ Terminal Emulators
  - [dua](https://github.com/Byron/dua-cli) - View disk space usage and delete unwanted data, fast. [https://lib.rs/crates/dua-cli](https://lib.rs/crates/dua-cli)
  - [dust](https://github.com/bootandy/dust) - A more intuitive version of du in rust.
  - [eureka](https://github.com/simeg/eureka) - 💡 CLI tool to input and store your ideas without leaving the terminal.
- - [exa](https://github.com/ogham/exa) - A modern version of ‘ls’. [https://the.exa.website/](https://the.exa.website/)
+ - [eza](https://github.com/eza-community/eza) - A modern version of ‘ls’. [https://eza.rocks/](https://eza.rocks/)
```

Comment:
Keep exa and add the [DEPRECATED] text as it is added in the rest of the repo (I would do it but not sure how to modify your PR)

Reply:
Resolved in ee831fbbc161527456820e2c0d99787485518116